peps/pep-0741.rst

```diff
@@ -5,15 +5,14 @@ Discussions-To: https://discuss.python.org/t/pep-741-python-configuration-c-api-
 Status: Draft
 Type: Standards Track
 Created: 18-Jan-2024
-Python-Version: 3.13
+Python-Version: 3.14
 Post-History: `19-Jan-2024 <https://discuss.python.org/t/pep-741-python-configuration-c-api/43637>`__,
               `08-Feb-2024 <https://discuss.python.org/t/pep-741-python-configuration-c-api-second-version/45403>`__,
 
 Abstract
 ========
 
-Add a C API to the limited C API to configure the Python initialization.
-It can be used with the stable ABI.
+Add a C API to configure the Python initialization.
```

Comment:
We have (more than) one of these already. It's worth saying in the first sentence why this one is special (i.e. "without relying on C structures" or "with the ability to make ABI-compatible changes in the future")

Reply:
done